### DISCUSSION TITLE
Problems about deepq.load

### DISCUSSION THREAD

**Discussion by PaullyWu** [bulletphysics/bullet3#3602] [General]
It seems like that function 'deepq.load' has been replaced with 'deep.learn' in the new version of "baselines", but some programs haven't been updated, resulting in some errors. For example, "enjoy_pybullet_cartpole.py" still uses 'deep.load'

**Comment by erwincoumans** [comment]
It is unfortunate that OpenAI keeps on renaming methids in the gym API. We probably need to stick to a specific gym version, document that gym version in a requirements file, and bump it up if a volunteer shows up. (same for tensorflow etc)

**Comment by PaullyWu** [comment]
I am very instersted in this project. So I'm considering that if you need a part-time volunteer for this issue?
